frontend/src/lib/constants.tsx

```diff
@@ -71,7 +71,7 @@ export const FEATURE_FLAGS = {
     FEEDBACK_CALL_CTA: 'feedback-call-cta', // owner: @paolodamico
     // Experiments / beta features
     INGESTION_GRID: 'ingestion-grid-exp-3', // owner: @kpthatsme
-    FUNNEL_HORIZONTAL_UI: '5730-funnel-horizontal-ui', // owner: @alexkim205
+    FUNNEL_HORIZONTAL_UI: '5730-funnel-horizontal-ui', // owner: @alexkim205 `control`, `test`
```

Comment:
Q: Did you delete the FF, so you could create the experiment with the same FF?

Reply:
Yup I deleted the FF and recreated it with a new experiment. Is that the recommended way to use it? I didn't think I should create a separate FF for the same feature

Reply:
Yep reasonable